bundle/phases/deploy.go

```diff
@@ -143,23 +143,25 @@ func deployCore(ctx context.Context, b *bundle.Bundle) {
 	}
 }
 
+// uploadLibraries uploads libraries to the workspace.
+// It also cleans up the artifacts directory and transforms wheel tasks.
+// It is called by only "bundle deploy".
+func uploadLibraries(ctx context.Context, b *bundle.Bundle, libs map[string][]libraries.ConfigLocation) {
+	bundle.ApplySeqContext(ctx, b,
+		artifacts.CleanUp(),
+		libraries.Upload(libs),
+	)
+}
+
 // deployPrepare is common set of mutators between "bundle plan" and "bundle deploy".
-// Ideally it should not modify the remote, only in-memory bundle config.
```

Comment:
Should we live this as a constraint: "Does not modify the remote, only in-memory bundle config".

I'd also add explanation of return value.,